tests/e2e/util_test.go

```diff
@@ -202,3 +210,111 @@ func ChmodFile(pathFile string, permission os.FileMode) {
 		}
 	}
 }
+
+func CheckDefaultStorageClass() error {
+	storageClass := kc.Resource("sc")
+	res := kubectl.List(storageClass, kc.GetOptions{Output: "json"})
+	if !res.WasSuccess() {
+		return fmt.Errorf(res.StdErr())
+	}
+
+	defaultStorageClassFlag := false
+	var scList storagev1.StorageClassList
+	err := json.Unmarshal([]byte(res.StdOut()), &scList)
+	if err != nil {
+		return err
+	}
+
+	for _, sc := range scList.Items {
+		isDefault, ok := sc.Annotations["storageclass.kubernetes.io/is-default-class"]
+		if ok && isDefault == "true" {
+			defaultStorageClassFlag = true
+			break
+		}
+	}
+	if !defaultStorageClassFlag {
+		return fmt.Errorf(
+			"Default StorageClass not found in the cluster: please provide a StorageClass name or set a default StorageClass.",
+		)
+	}
+	return nil
+}
+
+func ipToInt(ip net.IP) (result uint32) {
+	for i := 0; i < 4; i++ {
+		result |= uint32(ip[i]) << (24 - 8*i)
+	}
+	return
+}
+
+func intToIP(ipInt uint32) (result net.IP) {
+	result = net.IPv4(
+		byte(ipInt>>24),
+		byte(ipInt>>16),
+		byte(ipInt>>8),
+		byte(ipInt),
+	)
+	return
+}
```

Comment:
Are you sure about these functions? I don't fully understand how they work, leave comments